### ISSUE TITLE
Installing development version for glueviz with conda from source.

### ISSUE THREAD

**Issue by parampatidar** [glue-viz/glue#2178]
If I go through installation process for latest development version as described [here](http://docs.glueviz.org/en/stable/installation/developer.html#from-source-if-you-use-conda) to pull in all the dependencies.
The command `conda remove glueviz` should remove only `glueviz` package but it removes all the dependencies that conda `conda install -c glueviz glueviz` command pulls. 

But If I use `conda remove --force glueviz` as conda documentation says:

```
 --force            Forces removal of a package without removing packages
                    that depend on it. Using this option will usually
                    leave your environment in a broken and inconsistent
                    state
```
It only removes `glueviz` as we want.

**Comment by dhomeier** [comment]
Yes, that appears to be odd behaviour on the side of conda, since `glueviz` is a meta-package that depends on those other packages, and not the other way around. That said, since it does not install anything by itself, I don't see a strong reason to remove it from the development environment. I am wondering if the docs rather meant to say `conda remove glue-core`, analogously to `glue-vispy-viewers` below. But even that does not seem to be necessary, since pip will automatically uninstall the conda version first.

**Comment by astrofrog** [comment]
Yeah, it might actually make sense to modify the instructions to just start with an empty environment and pip install the dev version in there.
